src/components/Navbar.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from "react";
```

Comment:
Navbar shoulde be sticky, scrolling down hides this. Makes it difficult to search